src/Core/Manifest/PrioritySorter.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+namespace SilverStripe\Core\Manifest;
+
+use SilverStripe\Core\Injector\Injectable;
+
+/**
+ * Class PrioritySorter
```

Comment:
Can you document the purpose of this class here and maybe give an example usage? we don't use @package anymore.